### PR TITLE
Add a new package called mp_pytorch from pypi

### PR DESCRIPTION
**Pull request by Delashabee**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Delashabee** [comment]
@conda-forge/help-python The recipes are ready for review.THX :)

File: recipes/mp_pytorch/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    - mp_pytorch
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/ALRhub/MP_PyTorch
+  summary: The Movement Primitives Package in PyTorch
+  license: GPL-3.0
```
Comment (jaimergp) [inline recipes/mp_pytorch/meta.yaml:39]:
```suggestion
  license: GPL-3.0-only
```

Comment (jaimergp) [reply recipes/mp_pytorch/meta.yaml:39]:
```suggestion
  license: GPL-3.0-or-later
```

Comment (jaimergp) [reply recipes/mp_pytorch/meta.yaml:39]: Choose the right one :D

File: recipes/mp_pytorch/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  run:
+    - python >=3.6
+    - numpy
+    - matplotlib-base
+    - pytorch
+    - addict
+
+test:
+  imports:
+    - mp_pytorch
```
Comment (jaimergp) [inline recipes/mp_pytorch/meta.yaml:30]: Can you add a few more imports here? Their `__init__` is only defining `__version__` so it's not covering much.

For example:

```suggestion
    - mp_pytorch
    - mp_pytorch.mp
    - mp_pytorch.demo.demo_basis_gn
```

**Comment by Delashabee** [comment]
@conda-forge/help-python Hi!The recipes are ready for review. :)

**Comment by Delashabee** [comment]
**Hello,this recipes are ready to be reviewed ! @conda-forge/staged-recipes**

**Comment by Delashabee** [comment]
@conda-forge/help-python @conda-forge/staged-recipes Hi!The recipes are ready for review. If you encounter any problems, please check and let me know. I sincerely hope you can check my pull request, because it is very important to me and my laboratory.

**Review by xhochy — APPROVED** [review]

**Comment by xhochy** [comment]
If CI keeps passing on Linux, I'll merge.

**Comment by Delashabee** [comment]
> If CI keeps passing on Linux, I'll merge.

Thx!

**Comment by Delashabee** [comment]
Thx , what else do I need to do?

**Comment by Delashabee** [comment]
@conda-forge/help-python @conda-forge/staged-recipes Hi, I would like to add my tutor,who is also the creater of mp_pytorch, as the maintainer. How could I do it?

**Comment by xhochy** [comment]
Use the bot command `@conda-forge-admin, please add user @...` on the feedstock repository.

**Pull request merged**